code/modules/antagonists/ninja/ninja.dm

```diff
@@ -29,7 +29,7 @@
 	antag_memory += "Surprise is my weapon. Shadows are my armor. Without them, I am nothing. (//initialize your suit by clicking the initialize UI button, to use abilities like stealth)!<br>"
 	antag_memory += "Officially, [helping_station?"Nanotrasen":"The Syndicate"] are my employer.<br>"
 
-/datum/antagonist/ninja/proc/addObjectives(quantity = 6)
+/datum/antagonist/ninja/proc/addObjectives(quantity = 3)
```

Comment:
Why did we change this

Reply:
Ninjas were getting a longer list than what's possible to choose from, resulting in ninja objectives saying things like 1: Succeed. Success!